index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header class="header_container">
+        <nav class="navbar">
+            <h2>Welcome</h2>
+            <img src="/assets/Union.png" alt="bg">
+        </nav>
+        <section class="intro_section">
+            <h2> Hey there. I'm Favour <br>
+                <span>I'm a software developer</span></h2>
+            <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        </section>
+        <div class="social-icon">
+            <a href="www.facebook.com"><i class="fa fa-facebook" aria-hidden="true"></i></a>
+            <a href="www.twitter.com"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            <a href="www.github.com/fayob"><i class="fa fa-github" aria-hidden="true"></i></a>
+            <a href="www.linkedin.com/in/abimbola-ade" class="fa fa-linkedIn"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+        </div>
```

Comment:
Consider changing the `div` tag to a `ul` tag. Since the list of icons is a list, obviously 😊, it would make more sense if this is an unordered list with each icon as a list item wrapped in `li`.